src/pages/VenuePage/TemplateWrapper.tsx

```diff
@@ -101,6 +102,12 @@ const TemplateWrapper: React.FC<TemplateWrapperProps> = ({ venue }) => {
       template = <FireBarrel />;
       break;
 
+    case VenueTemplate.embeddable:
+      template = <Embeddable venue={venue} />;
+      fullscreen = true;
+
+      break;
+
```

Comment:
Ideally these cases would be sorted alphabetically.

Reply:
We can cast the venue type to the correct 'specific type' here, rather than passing through the generic `Venue` type then hacking it within the component itself.

Reply:
'specific type' fixed in 7c16f06 (enabled by fdb720c + 03a9443)

'sort order' fixed in https://github.com/sparkletown/sparkle/pull/1243/commits/df631f4be96346b9b8d485322d441cac73156a88